operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -768,9 +776,12 @@ distributor:
     enabled: false
     minReplicaCount: 1
     maxReplicaCount: 10
-    pollingInterval: 10
+    # -- Setting pollingInterval here will override the global value
+    # pollingInterval:10
     targetCPUUtilizationPercentage: 100
     targetMemoryUtilizationPercentage: 100
+    # -- if empty global setting will be used
+    prometheusAddress: ""
```

Comment:
Do you think there's real use case for a per-compoentn prometheus address? Would you scale the distributor using different prometheus installation than the querier? It might simplify the code if we keep just one global `kedaAutoscaling`. It's similar for pollingInterval - my preference is to keep the implementation simpler if there's no apparent need for this right now. WDYT?

Reply:
> Do you think there's real use case for a per-compoentn prometheus address? Would you scale the distributor using different prometheus installation than the querier? It might simplify the code if we keep just one global `kedaAutoscaling`. It's similar for pollingInterval - my preference is to keep the implementation simpler if there's no apparent need for this right now. WDYT?

You're probably right. I had our setup in mind. We don't use metamonitoring, but a self-installed Prometheus operator. I came from the point of view that people might want to migrate between metamonitoring, their own Prometheus or other solutions. To keep the impact as small as possible when migrating the monitoring, I thought it would make sense to be able to customize individual components. But you are right, probably nobody will use this, or if they are faced with this use case, they will know how to help themselves otherwise. 

I would then implement your suggestion and implement the simpler variant, with just one global section.

Reply:
@dimitarvdimitrov Do you think a per component custom header setting makes sense?